mlflow/models/evaluation/base.py

```diff
@@ -673,6 +847,8 @@ def evaluate(
     evaluators=None,
     evaluator_config=None,
     custom_metrics=None,
+    validation_thresholds=None,
+    baseline_model=None,
 ):
     """
```

Comment:
@jerrylian-db Can we add some documentation to `mlflow.evaluate()` (outside of the params section) explaining how this API can be used for model validation?